### ISSUE TITLE
Trouble with the overhangs for the impeller

### ISSUE THREAD

**Issue by RebelDesigner** [Makerr-Studio/CentrifugalDustSeparator-#7]
![image](https://github.com/user-attachments/assets/71616794-a89a-4092-916b-9c1d3bdf3c7d)

Hi,
Printing with PLA on an A1 and the impeller is just really hard to print for me.
Used the Bambu 3mf files and speeds.
This is my second attempt now with lower speeds.
Thinking of printing this also with even lower speeds on the overhangs (now the defaults for slowing down during overhangs).
Did anyone else have such a hard time printing this?

**Comment by Makerr-Studio** [comment]
I made a short about that topic;
https://m.youtube.com/shorts/qC6KJh5NBew

I printed the first one in januari when it was colder. 
now I need to print with the door open en top cover removed. The normal impeller printed fine on my X1 carbon. It was the mirrored impeller that caused issues.

**Comment by RebelDesigner** [comment]
I started to suspect room temperature as well. The printer was in a closed off room all night and the graph of an external temperature sensor is pretty much in line with the ugly overhangs. At a certain point the temperature starts to drop and the overhangs gradually start to look better.

I will wait a few days until the temperature starts to drop here so I hope it results in a better print!
Thanks!

**Comment by RebelDesigner** [comment]
I cannot get the impeller to print correctly.
Have tried everything. Printing when the ambient temperature was lower, and even an airconditioned room at 18 degrees.
Slowing down to 50% printing speed on the A1 also does not improve to a perfect result.

Any tips I can try?

**Comment by RebelDesigner** [comment]
I have not. It was a new spool straight from the vacuum packaging.

**Comment by RebelDesigner** [comment]
I bought a dryer and dried it. Indeed, also with a new spool this time, the result is much better. It’s not perfect yet. The front side is now smooth, yet the back (so the actual overhang) is still a bit of a moon landscape. I think next I’m going to install the prusa slicer and see if I can get a decent quality print of the mirrored impeller. Maybe the slicer also makes a difference.

Btw I printed with overhang speed of 30mm/s. That seems to give the best results.

**Comment by asucks** [comment]
What quality settings are you using in Bambu Slicer?  I would recommend switching the print order to "outer/inner" if you haven't already.  This will fix a lot of those issues.

You can also mess around with "Arachne" wall generator settings, as that can usually improve more detailed overhangs/curves in the slicer.

I would continue to slow down as well. 

<img width="450" alt="Screenshot 2024-09-08 at 10 42 41 PM" src="https://github.com/user-attachments/assets/8fa7c4ab-954f-4849-8f0e-5675afd0f430">

**Comment by Makerr-Studio** [comment]
Did you try the OrcaSlicer and clockwise printing already?

**Comment by RebelDesigner** [comment]
I have been printing other stuff yesterday so I haven’t had time to play with OrcaSlicer yet. I will try tonight.

Thanks for the tips regarding the order of walls. I will also do a test print with that setting later.

**Comment by RebelDesigner** [comment]
Tried switching the order of walls as well as the Arachne wall generator. Unfortunately no real improvements.

I have one pretty decent print (slowing down the overhangs to 30mm/s, but slowing down the entire print to 50%) where only the overhanging side of the fins are a bit like sandpaper, but the top side is looking OK. I’m going to sand that back side a bit and call it good. Next up today will be the Orca Slicer for the mirrored impeller.

I think in general this design can only be printed on the really high end machines. An A1 is just not good enough. Not sure if the design can be changed for easier printing without changing its characteristics.

**Comment by Makerr-Studio** [comment]
If you still can’t get the impeller smooth I can print them for you. I believe you live in the Netherlands so sipping will not be a problem. Are you making the EU version?

**Comment by RebelDesigner** [comment]
That is nice of you to offer! I do live in NL (Breda area), so I might take you up on your offer. Thank you.

I tried the Prusa Slicer with the mirrored impeller and the direction reversed. Had to stop the print after two hours because the quality was really bad again. Looking at the design though, is there a reason for the fins to be about 1mm thick on the bottom while being almost 3mm thick on the top? Making the fins thicker might help with stability while printing?

**Comment by Makerr-Studio** [comment]
I did not designed the impellor blades myself. It is a 1:1 copy from the original file from pilson guitars. So a lot of people did print it this way. I only made the mirrored version of it 😅.

**Comment by RebelDesigner** [comment]
@Stevenpolak both versions. Up until yesterday I only tried printing the non-mirrored version. Both have problems.

@Makerr-Studio if the offer still stands to print them for me I would like to take you up on that! Let me know what the best way is to get in touch.

Still find it weird that I am having such a hard time printing this. I must be doing something wrong. Or the A1 is just not capable of printing such a difficult part.

**Comment by Makerr-Studio** [comment]
Yes it still stand. You can contact me on info@capturingdust.com.

**Comment by cewalker7** [comment]
![IMG_6955](https://github.com/user-attachments/assets/4d629723-edc6-414b-ac85-e45181c5cfaf)
I am having the same issue. I have tried various settings   Any settings advice?  I have also tried opening my P1S and putting a box fan pointing in it.

**Comment by cewalker7** [comment]
![IMG_6957](https://github.com/user-attachments/assets/3809fc4d-4a52-4aa2-a11f-c28e7d4e4665)
![IMG_6956](https://github.com/user-attachments/assets/6c34bce7-7b86-4a59-941d-3b98ad236528)
I am having the same issue. One side looks better but the overhang part not so much.  I have tried various settings   Any settings advice?  I have also tried opening my P1S and putting a box fan pointing in it.

**Comment by jmichas** [comment]
I am adding my settings for my P1S that have been working well so far. I am about 1.5h into my 8h print. Previously I had the same rough issues that others had very early in the print so I aborted.

I started with the posted 3mf and made changes.

So my changes are (using orca slicer):
Wall Order: Outer/inner
Wall Loop Direction: Clockwise

Speed:
First Layer: 30
First Layer Infill: 50
Overhangs: 
first box (10%,25%): 30
second box (25%,50%) : 20

Filament Settings for Bambu PLA Matte, which is what I am using:
Nozzle temp: 225/225
Textured PEI Plate 58/58

Cooling:
no cooling for first 3 layers
part cooling fan 100%/100%
Aux fan speed 100%

after first 3 layers are printed and I hear the fans start I also bumped the chamber fan to 90%

So far everything is super smooth, not even a booger anywhere.

Fingers crossed....
